lib/containers.js

```diff
@@ -52,32 +60,48 @@ module.exports = {
                         .replace('$ID', container.id);
 
                     container.log_stream = logStream.replace('$LOG_URL', url);
-                    container.logs = [];
+                    container.logs;
```

Comment:
unsetting this as a variable, so we know the difference between failed attempts and empty logs from docker server.